### ISSUE TITLE
RIDE should survive if configuration file is corrupted

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#990]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=952) by @mkorpela on 15 Mar 2012*

Currently RIDE can be started only after removing the corrupted configuration file.

This issue is reported to us now and then so possibly it is a large problem - and most likely (yes I'm just guessing) the users that encounter it will never use RIDE again.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=952#c2) by @mkorpela on 15 Mar 2012*


Fixed after <a href="https://github.com/robotframework/RIDE/commit/63bd01252c77fe1b952c114113f910fb47c87aba" rel="nofollow">https://github.com/robotframework/RIDE/commit/63bd01252c77fe1b952c114113f910fb47c87aba</a>

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=952#c3) by @mkorpela on 10 Jun 2012*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=166" title="Show error if user settings are invalid"> Issue 166 </a> has been merged into this issue.
